src/services/accounts/AccountsStakingPayoutsService.ts

```diff
@@ -152,6 +152,7 @@ export class AccountsStakingPayoutsService extends AbstractService {
 		const startEra = Math.max(0, sanitizedEra - (depth - 1));
 		const runtimeInfo = await this.api.rpc.state.getRuntimeVersion(at.hash);
 		const isKusama = runtimeInfo.specName.toString().toLowerCase() === 'kusama';
+		const stakingVersion = await historicApi.query.staking.palletVersion<u16>();
```

Comment:
This is an awesome call. Has this been around for a bit? Is it safe to use historically?

Reply:
Well, it has been around since v13. Before that it was `api.query.staking.storageVersion()` and it returned a type `Releases` first and then `PalletStakingReleases`, that went from `V0` to `V12_0_0`